include/invert_quda.h

```diff
@@ -521,6 +521,9 @@ namespace quda {
     bool recompute_evals;   /** If true, instruct the solver to recompute evals from an existing deflation space. */
     std::vector<ColorSpinorField *> evecs; /** Holds the eigenvectors. */
     std::vector<Complex> evals;            /** Holds the eigenvalues. */
+    ColorSpinorField dummy; /** dummy addition for now */
```

Comment:
Can we avoid that somehow?

Reply:
This is slightly better: 7bb63aabceacf48d1dd2c1c42a7ea536bf1e7b98